### ISSUE TITLE
No warnings (or errors) when using `"severity": "warning"` for rule: `comment-format`

### ISSUE THREAD

**Issue by mischah** [palantir/tslint#3928]
### Bug Report

- __TSLint version__: 5.10.0
- __TypeScript version__: 2.8.3
- __Running TSLint via__: CLI
  - ```shell
    tslint --project tsconfig.json -c tslint.json -t stylish "src/**/*.ts"

#### TypeScript code being linted

```ts
// console.log('Show last project:');
// console.log(JSON.stringify(projects.pop(), null, 2));
```

with `tslint.json` configuration:

```json
{
  "extends": [
    "tslint-xo/space",
    "tslint-config-prettier"
  ],
  "rules": {
    "comment-format": {
      "severity": "warning"
    }
  }
}
```

with `tsconfig.json` configuration:

```json
{
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
     "lib": [
         "es5",
         "es6",
         "es7"
      ],
      "target": "es5",
      "module": "commonjs",
      "moduleResolution": "node",
      "outDir": "./build",
      "emitDecoratorMetadata": true,
      "experimentalDecorators": true,
      "sourceMap": true,
      "sourceRoot": "./src",
      "strictNullChecks": true
   }
}
```

#### Actual behavior

No error (and no warning) regarding the rule `comment-format`.

It behaves like the rule is switched off.

#### Expected behavior

Showing a warning like:
```shell
path/to/myFile.ts:1:3
WARNING: 1:3   comment-format  comment must start with uppercase letter
WARNING: 2:3   comment-format  comment must start with uppercase letter
```

cc/ @nicolaisueper

**Comment by giladgray** [comment]
@mischah i think you also need to specify some format options? my hunch is that by default the rule has no config so it essentially behaves like it's off. try adding `options: ["check-lowercase"]` to the config.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
